tests/Dialect/ModArith/Conversions/mod_arith_to_arith/runner/lower_add.mlir

```diff
@@ -17,4 +25,4 @@ func.func @test_lower_add() {
   return
 }
 
-// CHECK_TEST_ADD: [1225, 7276, 7645, 0]
+// CHECK_TEST_ADD: [1258, 7276, 7678, 0]
```

Comment:
I was actually confused by the original runner result. The original result is not correct for either signed interpretation (this PR's semantic) or unsigned interpretation (original semantic).

For example, `67108862 = -2` and `67108863 = -1` for i26, yet there addition result modulo 7681 should not be 7645.

```python
>>> (67108863 + 67108862) % 7681
7612
>>> (-1 + -2) % 7681
7678
```

Though the first result is correct: we have `36789492 = -30319372` and `29498763`,

```python
>>> (36789492 + 29498763) % 7681
1225
>>> ((-30319372) + 29498763) % 7681
1258
```


Then the first result is incorrect for `lower_mul.mlir`, 

```python
>>> (36789492 * 29498763) % 7681
1637
>>> ((-30319372) * 29498763) % 7681
1600
```

Reply:
Oh..in the original semantic, mod_arith.add/sub/mul require the input to be in [0, q), and the runner tests violated that constraint.